test-utilities/src/main/kotlin/io/github/apl_cornell/viaduct/TestFileProvider.kt

```diff
@@ -7,6 +7,12 @@
 import java.util.stream.Stream
 import kotlin.streams.asStream
 
+/** Enumerates the paths of source files that use the new IR and should successfully compile. */
```

Comment:
Move this class into `ParsingPrintingText`.

Reply:
The class uses a function which is private to `TestFileProvider`. Should I make it public?

Reply:
Ugh, that's right. Just leave the class where it is. We need to reorganize test cases anyway.